code/datums/elements/effect_trail.dm

```diff
@@ -3,14 +3,17 @@
  */
 
 /datum/element/effect_trail
+	element_flags = ELEMENT_BESPOKE
+	argument_hash_start_idx = 2
 	/// The effect used for the trail generation.
-	var/obj/effect/chosen_effect
+	var/chosen_effect
 
-/datum/element/effect_trail/Attach(datum/target)
+/datum/element/effect_trail/Attach(datum/target, chosen_effect)
```

Comment:
```suggestion
/datum/element/effect_trail/Attach(
	datum/target,
	chosen_effect = /obj/effect/forcefield/cosmic_field,
)
```
There should have a base one in case it's not set, prevents things like admins breaking it because they can give components/elements to anything.